lib/state/ui/reducer.ts

```diff
@@ -52,19 +63,6 @@ const simperiumConnected: A.Reducer<boolean> = (state = false, action) =>
     ? action.simperiumConnected
     : state;
 
-const visiblePanes: A.Reducer<string[]> = (
-  state = defaultVisiblePanes,
-  action
-) => {
-  if ('TAG_DRAWER_TOGGLE' === action.type) {
-    return action.show
-      ? union(state, ['tagDrawer'])
-      : difference(state, ['tagDrawer']);
-  }
-
-  return state;
-};
```

Comment:
Thanks for removing this. Probably would have been nice to do it in a separate PR with its own traceable history, or at least a mention in the `git` commit, since now we don't have any explanation for why this was removed in the code or `git` history - it just disappears.